onboarding_notes/noelle.md

```diff
@@ -0,0 +1 @@
+Hiiiiii!
```

Comment:
What is the purpose of this file? Even if we don't want to fully flesh out the file contents yet in this CL can we give it more descriptive contents? 

Eg)

Documentation for Foo.

Under construction <link to corresponding issue>

Reply:
This is under construction. @no0ro is working on building out an onboarding doc for future bootcampers who join the project. The doc will have information on how to merge/rebase, how to make PRs, etc...